### ISSUE TITLE
dependency error on vite 6

### ISSUE THREAD

**Issue by steffchep** [vite-pwa/vite-plugin-pwa#800]
I get this:

```
Could not resolve dependency:
npm error peer vite@"^3.1.0 || ^4.0.0 || ^5.0.0" from vite-plugin-pwa@0.21.0
npm error node_modules/vite-plugin-pwa
npm error   vite-plugin-pwa@"^0.21.0" from the root project
```

When trying to npm install with vite 6
Workaround is to use --force or --legacy-peer-deps, but please add vite 6 :)

*Edit* I see there is a PR for this! 👍

**Comment by gabrielrbarbosa** [comment]
#797

**Comment by userquin** [comment]
I'm going to release a new minor (v0.22.0) adding vite 6 to the peers + fix build when manifest enabled via #797 PR : we're going to keep #786  (using Vite 6 Environment API) open for a while, we want to do some more tests.

**Comment by userquin** [comment]
closed via #797

**Comment by franciscop** [comment]
This is a follow-up fix to allow for `npm create @vite-pwa/pwa {NAME} -- --template react`:

https://github.com/vite-pwa/create-pwa/pull/32

Is there other recommended way that I might be missing?

**Comment by userquin** [comment]
I need to merge your pr and release a new version for react, I'm very busy, I'll try to release the fix next weekend (sorry 🙏  ).

**Comment by franciscop** [comment]
Thanks! No worries, I fixed it locally for myself, this PR was for the community so others can benefit! 🙇‍♂️🙇‍♂️

**Comment by userquin** [comment]
Your PR merged and released `v0.6.2`, thx ❤
